app/controllers/help_controller.rb

```diff
@@ -1,10 +1,5 @@
 class HelpController < ApplicationController
 
```

Comment:
here's a bookmark to make it easier to find the help_controller:

note I renamed the views   :   
help > old_help
en_help > help

so that this controller just renders the help doc